Gaijinizer patch/CommonEvents_670.txt

```diff
@@ -3,214 +3,270 @@
+> CONTEXT: CommonEvents_670/91/Dialogue/kazuya-0
+Aa... ♡♡ AAAAAAAHHHH...! ♡
 
+Nooo...... ♡
+I can't...... hold baaack... ♡#Strate
 > END STRING
 
 > BEGIN STRING
 \\N[1]はアナオンの口の中に精を注ぎ込んでしまった…
-> CONTEXT: CommonEvents_670/161/Dialogue < UNTRANSLATED
-
+> CONTEXT: CommonEvents_670/161/Dialogue
+\\N[1] poured his semen into Anaon's mouth...#Strate
 > END STRING
 
 > BEGIN STRING
 ああ………っ♡♡イイっ……♡
 
 すごく美味しい………♡
-> CONTEXT: CommonEvents_670/163/Dialogue/enemy33 < UNTRANSLATED
+> CONTEXT: CommonEvents_670/163/Dialogue/enemy33
+Aahh...... ♡♡ So good...... ♡
 
+It's delicious...... ♡#Strate
 > END STRING
 
 > BEGIN STRING
 フフフッ♡♡
 人間の精気は、やっぱり精液から摂取するのが一番ね…♡
 
 血肉に比べて、お肌も更にツヤツヤになるし……♡♡
-> CONTEXT: CommonEvents_670/167/Dialogue/enemy33 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_670/167/Dialogue/enemy33
+Fufufu♡♡ I guess the best way to absorb human
+essence really is through their semen, huh...? ♡
+Compared to their flesh, it makes your skin
+shinier...... ♡♡#Strate
 > END STRING
 
 > BEGIN STRING
 最近は、ティターニアの配下の妖精たちに
 迷いこんだ人間を獲られっぱなしだったから……
 
 果物以外の食事は、本当に久しぶり……♡♡
-> CONTEXT: CommonEvents_670/172/Dialogue/enemy33 < UNTRANSLATED
-> CONTEXT: CommonEvents_702/231/Dialogue/enemy33 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_670/172/Dialogue/enemy33
+> CONTEXT: CommonEvents_702/231/Dialogue/enemy33
+Lately, the fairies under Titania have been
+capturing all the humans that wandered in here...
+It's been so long since I've eaten anything other
+than fruits......♡♡#Strate
 > END STRING
 
 > BEGIN STRING
 まぁ、これで３０年は食べ物には困らないわね♡
 
 １滴も出なくなるまで搾ってあげる……♡
-> CONTEXT: CommonEvents_670/177/Dialogue/enemy33 < UNTRANSLATED
+> CONTEXT: CommonEvents_670/177/Dialogue/enemy33
+Well, now I won't have to worry about food for
+the next 30 years♡
 
+I'll squeeze you to the last drop...... ♡#Strate
 > END STRING
 
 > BEGIN STRING
 フフフッ、あなたは知らないかもしれないけど……
 
 この『妖精の国』では、ニンゲンは
 \\C[2]寿命以外の方法で死ぬことはないのよ
-> CONTEXT: CommonEvents_670/217/Dialogue/enemy33 < UNTRANSLATED
+> CONTEXT: CommonEvents_670/217/Dialogue/enemy33
+Fufufu, perhaps you don't know this, but...
 
+In this "Fairy Realm", humans \\C[2]do not die
+from anything other than their own lifespan.#Strate
 > END STRING
 
 > BEGIN STRING
 だから、あなたは老いて射精ができなくなるまで
 毎日搾られ続けて、私の空腹を満たすの♡
 
 出なくなったら、丸呑みして、胎内で消化してあげる
-> CONTEXT: CommonEvents_670/222/Dialogue/enemy33 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_670/222/Dialogue/enemy33
+So, until you get so old that you can't ejaculate
+anymore, I'll keep squeezing you every day to
+satisfy my hunger♡ Then I'll swallow you whole
+and digest you inside my womb.#Strate
 > END STRING
 
 > BEGIN STRING
 まぁ、たまに果物を食べたり、ゆっくり眠ったり
 人並み程度の自由は許してあげるわ
 
 でもそれ以外、あなたの居場所は一生ここよ…♡
-> CONTEXT: CommonEvents_670/227/Dialogue/enemy33 < UNTRANSLATED
-
+> CONTEXT: CommonEvents_670/227/Dialogue/enemy33
+Well, I'll occasionally let you have some fruit,
+a good night's rest, and as much freedom as
+anyone else. But other than that, this is where
+you belong for the rest of your life... ♡#StrateUnsure
 > END STRING
```

Comment:
Note

Reply:
But would that really make sense? She's clearly not giving him as much freedom as a normal person. That's why I thought it tried imply more that she would give him as much freedom as "all the other humans", those being the humans captured by fairies (who don't have that much freedom themselves).

Reply:
人並 is generally used to mean "ordinary person" like, "I only have the power of an ordinary person", or "Living the life of an ordinary person." I think what she's meaning is other than the semen squeezing, she'll let him do "ordinary person" things like eating fruit occasionally and sleeping when he needs to. Those ったり at the end of the verbs imply that those actions are representative examples of the types of things she'll let him do, so there are probably other reasonable things she'll let him do as well.

Reply:
I see, I guess that makes more sense.
"Well, I'll occasionally let you have some fruit,
a good night's rest, and as much freedom as
a normal person. But other than that, this is
where you belong for the rest of your life... ♡"
How does that look? It's kind of a tight fit on the text box, otherwise I might have to rephrase the sentence a bit more thoroughly.

Reply:
Seems fine to me. If you need more space you can format across more than four lines to have the dialogue split between two text boxes.

Reply:
Looks good 👍